addons/website_slides/tests/test_ui_wslides.py

```diff
@@ -182,11 +182,7 @@ def test_course_reviews_elearning_officer(self):
             'groups_id': [(6, 0, (self.env.ref('base.group_user') | self.env.ref('website_slides.group_website_slides_officer')).ids)]
         })
 
-        # The user must be a course member before being able to post a log note.
         self.channel._action_add_members(user_demo.partner_id)
-        self.channel.with_user(user_demo).message_post(
-            body='Log note', subtype_xmlid='mail.mt_note', message_type='comment')
-
```

Comment:
why change test?

Reply:
It was possible to show the notes before in portal chatter, and it's not possible now. So there is no point in sending one in the test.

Reply:
Why the functionally change? Was it a bug? An oversight that wasn't used in practice?

Reply:
I think the test is still relevant, `group_website_slides_officer` looks like the kind of group that should be able to post a note in the chatter?

And be able to see notes as well.

I don't think we should hide notes in the portal for users that have normally access to them.
I think the spec of not showing them is only targeted towards portal users/that don't naturally have rights for them.

Reply:
In fact, it's not just for portal users. We had a long discussion with PO about this and she really wanted this spec. I return this part because it has no effect on the tests, but I think we need to rethink the spec of the task.